swift/downgrades/e02469b183f72c50d72f0c6926fdd8366ae45aec/upgrade.properties

```
@@ -0,0 +1,4 @@
+description: Remove new implicit conversion
+compatibility: partial
+unspecified_elements.rel: run unspecified_elements.ql
+implicit_conversion_exprs.rel: run implicit_conversion_exprs.ql
```

forgot about this when we chatted, we also need to delete the now non-existing table
```suggestion
implicit_conversion_exprs.rel: run implicit_conversion_exprs.ql
abi_safe_conversion_exprs.rel: delete
```